kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -260,7 +281,7 @@ object Fiber extends FiberPlatformSpecific:
       * @return
       *   A Fiber that completes with the result of the first Fiber to complete
       */
-    def race[E, A: Flat, S](
+    private[kyo] def race[E, A: Flat, S](
```

Comment:
I think having the same methods in `Async` and `Fiber` can be confusing so I've decided to remove some and only keep the ones needed to implement the `Async` methods as `private[kyo]`. It limits the functionality but it's also easy to go back to a `Fiber` via `Async.run`.